examples/hil.cfg

```diff
@@ -89,6 +89,19 @@ uri = sqlite:////absolute/path/to/hil.db
 # enabled:
 #dry_run=
 
+[maintenance]
+# Options for configuring the maintenance pool.
+#
+# Enables maintenance pool functionality if True. Default is False:
+# enable = False
```

Comment:
Might be better to just have this be implied by the presence or absence of the other two options.